integration-mc1-15/src/main/kotlin/org/eln2/ModBlocks.kt

```diff
@@ -1,11 +1,15 @@
 package org.eln2
 
 import net.minecraft.block.Block
+import net.minecraft.item.ItemGroup
 import org.eln2.blocks.FlubberBlock
+import org.eln2.blocks.OreBlock
+import org.eln2.utils.OreData
 
 /**
  * Blocks added here are automatically registered.
  */
 enum class ModBlocks(val block: Block) {
-	FLUBBER(FlubberBlock())
+	FLUBBER(FlubberBlock()),
+	ORE_NATIVE_COPPER(OreBlock(OreData(1.0f, "ore_native_copper", ModItems.ORE_CHUNKS_COPPER.items.defaultInstance)))
```

Comment:
Ideally, all blocks of the same 'type' will be variations of the same block using meta data. The total block ID count is still limited. At least, it was in 1.12; it's possible that 1.15 has changed something.

That's something I can refactor once we have >1 ore blocks, though.

Reply:
That sounds like the best approach.